software/homeserverhq.yaml

```
@@ -0,0 +1,10 @@
+name: "HomeServerHQ"
+website_url: "https://www.homeserverhq.com/"
+source_code_url: "https://github.com/homeserverhq/hshq"
+description: "HomeServerHQ is an all-in-one home server infrastructure and installer that embarrasses its competition. Have a fully configured email server, VPN, and public website(s) set up in less than an hour, even behind CGNAT. Is this Web 3.0? You decide."
```

A less boastful and more descriptive description.
```suggestion
description: "All-in-one home server infrastructure. It is designed to be simple and easy for anyone to setup and use, even without any prior software/IT experience."
```